SpringData/.classpath

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@habeeb-shaikh 
you can use gitignore file to stop committing IDE specific files, which are not required on repo.